docs/adguard-mini-for-mac/solving-problems/logs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How to collect logs in AdGuard Mini for Mac
+sidebar_position: 4
+---
+
+:::info
+
+This article is about AdGuard Mini for Mac, which safeguards only your Safari browser. To protect your Mac entirely, [download the AdGuard app](https://agrd.io/download-kb-adblock)
+
+:::
+
+To analyze and diagnose various problems that may occur with AdGuard Mini for Mac, the AdGuard support service may need its log files. These files contain records of errors and other software events. Below are brief instructions on how to collect logs and send them to support if necessary.
+
+### How to collect logs
+
+1. Open AdGuard Mini app.
+2. Go to *Settings*, scroll down and click *Export logs*.
+3. Choose a location to save the .zip file and click *Save*.
+
+Now you can send this file to support@adguard.com or attach it to your bug report on GitHub.
+
+:::note
+
+AdGuard is committed to protecting your privacy. We strictly follow our [Privacy Policy](https://adguard.com/privacy/mini-mac.html) and do not collect any private information about users. Before sending your logs to the support team, please review the file as it may contain additional information that you don’t want to share. If it contains such personal information, we recommend that you delete it first.
```

Comment:
this is the Privacy notice. besides, this link is not available yet. you can replace it with the link to our Privacy policy

Reply:
But maybe the link to the whole Privacy policy would be better better